release_bot/configuration.py

```diff
@@ -43,9 +45,29 @@ def __init__(self):
         self.github_app_installation_id = ''
         self.github_app_id = ''
         self.github_app_cert_path = ''
+
         self.clone_url = ''
         self.webhook_handler = False
 
+        # when you need to have different name for pypi release
+        self.pypi_project = ''
+
+
+    def pypi_project_from_setup_cfg():
```

Comment:
Forgot to a pass a `self` to the method. That's why the tests were failing.

`def pypi_project_from_setup_cfg(self):`